.env.example

```diff
@@ -10,7 +10,7 @@ REDIS_DSN=redis://localhost
 OSU_API_KEY=
 
 DOMAIN=cmyui.xyz
-MIRROR_URL=https://api.chimu.moe/v1
+MIRROR_URL=https://catboy.best/api
```

Comment:
why does the current most reliable beatmap mirror need to be named catboy.best

Reply:
is good

Reply:
> Can you add reminding that you can still use chimu ? 👉👈

You kinda took too long fixing it, so everyone started migrating to mino..

Reply:
> > Can you add reminding that you can still use chimu ? 👉👈
> 
> You kinda took too long fixing it, so everyone started migrating to mino..

tbh, its not easy task to move "our big dump" from ipfs to raw FS :/ That's one of the reason why it took sooo long. And now chimu should have 99.99% uptime :)